### PR TITLE
Feature/tri 668

### PR DESCRIPTION
**Pull request by ds-ext-kmassalski**
- AASHandler replaced by 4 delegates - each responsible for one domain,
- AASHandler code moved almost almost 1-1,
- added unit tests that were missing

### CODE REVIEW

File: irs-api/src/test/java/org/eclipse/tractusx/irs/aaswrapper/job/delegate/SubmodelDelegateTest.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+                        submodelDescriptor("urn:bamm:com.catenax.assembly_part_relationship:1.0.0#AssemblyPartRelationship",
+                                "testAssemblyPartRelationshipEndpoint"))));
+
+        // when
+        final ItemContainer result = submodelDelegate.process(itemContainerShellWithTwoSubmodels, jobParameterFilter(),
+                new AASTransferProcess(), "itemId");
+
+        // then
+        assertThat(result).isNotNull();
+        assertThat(result.getShells().get(0).getSubmodelDescriptors().size()).isZero();
```
Comment (ds-jhartmann) [inline irs-api/src/test/java/org/eclipse/tractusx/irs/aaswrapper/job/delegate/SubmodelDelegateTest.java:42]: Sonar marks this as minor code smell. Use .isEmpty() instead.

File: .run/IrsApplication.run.xml [hunk 2]
```diff
@@ -3,6 +3,7 @@
     <module name="irs-api" />
     <option name="SPRING_BOOT_MAIN_CLASS" value="org.eclipse.tractusx.irs.IrsApplication" />
     <option name="ACTIVE_PROFILES" value="local" />
+    <option name="ALTERNATIVE_JRE_PATH" />
```
Comment (ds-jhartmann) [inline .run/IrsApplication.run.xml:6]: What is this option for?

**Review by ds-jhartmann — APPROVED** [review]
Looks good to me. @ds-jkreutzfeld you might want to take a look at this as well.

File: irs-api/src/test/java/org/eclipse/tractusx/irs/aaswrapper/job/delegate/SubmodelDelegateTest.java (hunk 1 above)
Comment (ds-ext-kmassalski) [reply irs-api/src/test/java/org/eclipse/tractusx/irs/aaswrapper/job/delegate/SubmodelDelegateTest.java]: Done

File: .run/IrsApplication.run.xml (hunk 2 above)
Comment (ds-ext-kmassalski) [reply .run/IrsApplication.run.xml:6]: Intellij add its automatically on my env...

**Pull request merged**